packages/@aws-cdk/aws-scheduler-targets-alpha/lib/sns-publish.ts

```diff
@@ -0,0 +1,37 @@
+import { ISchedule, IScheduleTarget } from '@aws-cdk/aws-scheduler-alpha';
+import { Names } from 'aws-cdk-lib';
+import { IRole } from 'aws-cdk-lib/aws-iam';
+import * as sns from 'aws-cdk-lib/aws-sns';
+import { ScheduleTargetBase, ScheduleTargetBaseProps } from './target';
+import { sameEnvDimension } from './util';
+
+/**
+ * Use an Amazon SNS topic as a target for AWS EventBridge Scheduler.
+ */
+export class SnsPublish extends ScheduleTargetBase implements IScheduleTarget {
+  constructor(
+    private readonly topic: sns.ITopic,
+    private readonly props: ScheduleTargetBaseProps,
```

Comment:
```suggestion
    private readonly props: ScheduleTargetBaseProps = {},
```
Let's add a default value to simplify initialization when no props are needed.
Can you please then update all the `SnsPublish(topic, {})` to `SnsPublish(topic)` in this PR?

Reply:
Thanks for the advice! I have updated them.